### DISCUSSION TITLE
Expansion of appendix functionality for scientific papers

### DISCUSSION THREAD

**Discussion by blenback** [quarto-dev/quarto-cli#5169] [Feature Requests]
Hi, I'm currently on my first time using Quarto to write a scientific journal article. I can already see how it will save me a lot of time and streamline the revisions process however there is some additional functionality related to appendices that it would be great to have in the future. 

My request is similar to what is discussed here: #4581 in the sense that for my research field I often need appendices to:
**1.**  Come after the references section of the main text.  
**2.**  Have individual titles with a prefix including alphabetical sequencing e.g "Appendix A: title... "
**3.** For the individual figures and tables in the appendices to be sequenced with an alphabetical/numeric combination e.g Figure A1: title...  
**4.** For a seperate reference to list to follow after the appendices, containing only the citations used in the appendixes and with a seperate heading such as "Appendix references". 

For **1.** I can see that there are options/work-arounds. Either by adding a div `::: {#refs}
:::` as per:[https://quarto.org/docs/authoring/footnotes-and-citations.html#bibliography-generation](url) or by using the `appendix-style: none` YAML option: [https://community.rstudio.com/t/appendix-after-references-in-quarto/148895/5](url) 

For **2.** I have been able been able to achieve this using the `.appendix` class, however the YAML option:
 `crossref: 
appendix-delim: ":"` does not appear to be working, see this reprex: [SM_reprex.md](https://github.com/quarto-dev/quarto-cli/files/11213680/SM_reprex.md)

For **3.** there is a work-around using the Latex code from #4581:
`\appendix
\renewcommand{\thefigure}{A\arabic{figure}}
\renewcommand{\thetable}{A\arabic{table}}
\setcounter{figure}{0}
\setcounter{table}{0}`
However it would be nice to not have to include this in every subsequent paper I write and for this to achievable with a YAML option under `crossref:` such as such as `appendix-label` in a similar way as is done for sections.

Similarly for **4.** I can see that there are possible workarounds perhaps using this Pandoc filter:  [https://github.com/pandoc-ext/section-bibliographies#quarto](url). However, at the monent this only supports seperate reference lists for different sections under the same heading level and not specifically for the `.appendix` class.

 It would also be possible to split the appendices into a seperate .qmd file and then render together with the main body however according to #4581 this presents it's own problems and ultimately I would like to be able to work in just a single .qmd file to ease collaboration.

**Comment by turbanisch** [comment]
I'd love to see this, too. For PDF output, adding `{.appendix}` doesn't seem to do anything at all – references still show up *before* the appendix and the appendix is not marked as such. Adding raw LaTeX code such as `\appendix` doesn't seem to help either.
